web/pdf_attachment_viewer.js

```diff
@@ -116,13 +116,12 @@ class PDFAttachmentViewer {
   /**
    * @param {PDFAttachmentViewerRenderParameters} params
    */
-  render(params = {}) {
-    var attachments = params.attachments || null;
-    var attachmentsCount = 0;
+  render({ attachments, keepRenderedCapability = false, }) {
```

Comment:
`keepRenderedCapability` is not in the documentation for `PDFAttachmentViewerRenderParameters` above. Could you add it for consistency?